packages/ui/src/components/va-accordion/VaAccordion.stories.ts

```diff
@@ -1,12 +1,237 @@
+  await callback(element)
+  await sleep(ms)
+}
 
 export default {
   title: 'VaAccordion',
   component: VaAccordion,
+  tags: ['autodocs'],
+}
+
+export const Default = () => ({
+  components: { VaAccordion, VaCollapse },
+  setup () {
+    return {
+      modelValue: [false, false, false],
+      collapses: [
+        { title: 'Collapse', content: 'Content' },
+        { title: 'Collapse', content: 'Content' },
+        { title: 'Collapse', content: 'Content' },
+      ],
+    }
+  },
+  template: `
+    <va-accordion v-model="modelValue">
+      <va-collapse
+        v-for="(collapse, index) in collapses"
+        :key="index"
+        :header="collapse.title"
+      >
+        <div>
+          {{ collapse.content }}
+        </div>
+      </va-collapse>
+    </va-accordion>
+  `,
+})
+
+Default.play = async ({ canvasElement }) => {
+  const canvas = within(canvasElement)
+  const collapses = await canvas.getAllByRole('button', { name: 'Collapse' }) as HTMLElement[]
+  const [lastCollapse] = collapses.slice(-1)
+  collapses.push(lastCollapse)
+
+  for (const collapse of collapses) {
+    await DispatchEventWithDelay(collapse, userEvent.click, 500)
+  }
+}
+
+export const Stateful = () => ({
+  components: { VaAccordion, VaCollapse },
+  setup () {
+    return {
+      collapses: [
+        { title: 'Collapse', content: 'Content' },
+        { title: 'Collapse', content: 'Content' },
+        { title: 'Collapse', content: 'Content' },
+      ],
+    }
+  },
+  template: `
+    <va-accordion stateful>
+      <va-collapse
+        v-for="(collapse, index) in collapses"
+        :key="index"
+        :header="collapse.title"
+      >
+        <div>
+          {{ collapse.content }}
+        </div>
+      </va-collapse>
+    </va-accordion>
+  `,
+})
+
+Stateful.play = async ({ canvasElement }) => {
+  const canvas = within(canvasElement)
+  const collapses = await canvas.getAllByRole('button', { name: 'Collapse' }) as HTMLElement[]
+  const [lastCollapse] = collapses.slice(-1)
+  collapses.push(lastCollapse)
+
+  for (const collapse of collapses) {
+    await DispatchEventWithDelay(collapse, userEvent.click, 500)
+  }
 }
 
-export const Default = defineComponent({
-  components: { VaAccordion },
-  template: '<VaAccordion/>',
+export const Multiple = () => ({
+  components: { VaAccordion, VaCollapse },
+  setup () {
+    return {
+      modelValue: [false, true, false],
+      collapses: [
+        { title: 'Collapse', content: 'Content' },
+        { title: 'Collapse', content: 'Content' },
+        { title: 'Collapse', content: 'Content' },
+      ],
+    }
+  },
+  template: `
+    <va-accordion v-model="modelValue" multiple>
+      <va-collapse
+        v-for="(collapse, index) in collapses"
+        :key="index"
+        :header="collapse.title"
+      >
+        <div>
+          {{ collapse.content }}
+        </div>
+      </va-collapse>
+    </va-accordion>
+  `,
 })
+
+Multiple.play = async ({ canvasElement }) => {
```

Comment:
For collapse I would image we'd need interaction test for default and multiple collapses:

For default collapse:
* When I click on collapse - it opens.
* When I click on another collapse - that collapse opens and previous collapse closes.
* When I click on opened collapse - it closes.

For multiple collapse:
* When I click on collapse - it opens.
* When I click on another collapse - that collapse opens and previous collapse stays open.
* When I click on opened collapse - it closes.

Interactions 1 and 3 could be reused. 2nd is unique.